<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomFunctionDialog</class>
 <widget class="QDialog" name="CustomFunctionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custon Function</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbReturnType"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Return Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edtFunctionName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Function Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="chkPointer">
       <property name="text">
        <string>Pointer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddParam">
       <property name="text">
        <string>Add Param</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelParam">
       <property name="text">
        <string>DeleteParam</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Variable Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Variable Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Array Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pointer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Preview:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtPreview">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomFunctionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomFunctionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
